src/components/sidenav/sidenav.css

```diff
@@ -26,26 +26,23 @@
 }
 
 .item:hover {
-  background: #eee;
   font-weight: 600;
-  border-left: 4px solid #eee;
-  border-radius: 0 30px 30px 0;
-  width: 85%;
+  width: 100%;
+  color:#000;
+  background: #eee;
 }
 
 .active {
-  background: #eee;
-  border-left: 4px solid #FF6700;
-  border-radius: 0 30px 30px 0;
-  width: 85%;
+  width:100%;
```

Comment:
You can't put styles on `.active` class straight away. Because of this, the style of database cards in the create-project page have also got affected. Use something like `.sidenav .active` as style selector.